### PR TITLE
Update to Selenium 3.141.59 and HtmlUnit 2.33.3

### PR DESCRIPTION
**Pull request by RyanFromQA**
This required a slight rework of the configuration builder. I followed the process followed in Selenium's GridLauncherV3 to parse the arguments. All unit tests are passing and the compiled plugin installs and works in my environment.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,12 +15,12 @@
...
         <findbugs.failOnError>false</findbugs.failOnError>
 
-		<selenium.version>3.12.0</selenium.version>
-		<htmlunit.version>2.29.2</htmlunit.version>
+		<selenium.version>3.141.59</selenium.version>
+		<htmlunit.version>2.33.3</htmlunit.version>
 		<java.level>8</java.level>
 	</properties>
 	<artifactId>selenium</artifactId>
-	<version>3.12.1-SNAPSHOT</version>
```
Comment (Mobrockers) [inline pom.xml]: The plugin version will be updated by the maven release plugin. It should not be updated manually.

Comment (RyanFromQA) [reply pom.xml]: Reverted to previosu value

**Comment by sebastienbonami** [comment]
When will this be released? @Mobrockers

**Comment by lucasheringer** [comment]
I have the same question as @sebastienbonami. Need to update to 3.141.59 as it fixes one issues that I'm facing while using docker.

**Comment by Mobrockers** [comment]
Apologies. I do not actively work with selenium, jenkins or even Java anymore so I have little time to check these out.

**Pull request merged**